_projects/SP19-VirtualWaterSport.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: project
+title: Virtual Water Sport
+desc: |
+This is a sponsored project built in Unity and only supports HTC Vive.
+dl: https://drive.google.com/open?id=1Nve2zp-YRxdjGrZFz8w2r21xiZP4IAsI
```

Comment:
Please make this folder publicly accessible to anyone with the link

Reply:
Done.